install_config/master_node_configuration.adoc

```diff
@@ -1187,21 +1193,12 @@ the `auditFilePath` location in the *_/var/lib/origin_* or *_/etc/origin/master/
 auditConfig:
   auditFilePath: "/var/lib/origin/audit-ocp.log"
   enabled: true
-  maximumFileRetentionDays: 10
-  maximumFileSizeMegabytes: 10
-  maximumRetainedFiles: 10
+  maximumFileRetentionDays: 14
+  maximumFileSizeMegabytes: 500
+  maximumRetainedFiles: 15
 ----
 
-.Basic Audit for the Audit Log
-
-The directory *_/var/lib/origin_* will be created if it does not exist.
-
-You can specify advanced audit log parameters by using the following parameter
-value format:
-
-----
-openshift_master_audit_config={"enabled": true, "auditFilePath": "/var/lib/origin/audit-ocp.log", "maximumFileRetentionDays": 14, "maximumFileSizeMegabytes": 500, "maximumRetainedFiles": 5}
-----
+The directory *_/var/lib/origin_* is created if it does not exist.
```

Comment:
When? Should this be something like "When you configure the Audit function, the \*\_/var/lib/origin_* directory is created if it does not exist?"

Reply:
@anpingli Can you help with this question? 
I assume that the directory is created when the user updates the *_/etc/origin/master/master-config.yaml_* the *_/var/lib/origin_* directory is created. 
Also, must the user specify *_/var/lib/origin_* as the `auditFilePath`

Reply:
@mburke5678  Shall we use /var/log/origin/audit-ocp.log instead of /var/lib/origin/audit-ocp.log?  In my testing, the origin directory was created automatically.

Reply:
@anpingli 
1. This BZ seems to say that you cannot use `/var/log/origin` with Audit. 

2. It seems that you can specify any directory name here, `/var/lib/origin/mylogs.log` or `/etc/origin/master/mylong.log` as long as the path is correct? 

https://bugzilla.redhat.com/show_bug.cgi?id=1622044#c19

Reply:
@mburke5678  yes, we can use any directory.  /var/log/origin is a more common directory to store logs.

Reply:
@anpingli I made the change. PTAL